src/cli/utils/options.go

```diff
@@ -41,3 +41,27 @@ func ControlWithConfig(cfg config.RepoDetails) control.Options {
 
 	return opt
 }
+
+func BackupOptions() control.Backup {
```

Comment:
Nit: to make the purpose a little more clear.  `BackupOptions` sounds more like the name of a struct.
```suggestion
func DefaultBackupOptions() control.Backup {
```

Reply:
hmmm I'll work on the naming, not sure I want to go with `Default` in this function though since it does parse values from the passed in flags